integration_test/sql/migration.exs

```diff
@@ -339,6 +339,8 @@ defmodule Ecto.Integration.MigrationTest do
     :ok
   end
 
+  @moduletag capture_log: true
```

Comment:
Let's set `@tag capture_log: true` so we don't silence the whole test case by accident.